lib/src/arbiter.dart

```diff
@@ -60,3 +60,57 @@ class PriorityArbiter extends Arbiter {
     ]);
   }
 }
+
+/// Arbiter Roun Robin.
+class RoundRobinArbiterlogic extends Arbiter {
+  ///Mask to define pending requests to be attended
+  ///i.e. If bit 4 is granted in a 8 bit request, [requestMask] will be 11110000
+  ///turning off the bit granted along with all previous bits
+  List<Logic> requestMask = [];
+
+  ///Result of masking [_requests] with [requestMask]
+  ///Contains bits of [_requests] but as bits granted, will off
+  List<Logic> grantMask = [];
+
+  /// Initiliazing Round Robin
+  RoundRobinArbiterlogic(super.requests, Logic clk, Logic reset) {
+    clk = addInput('clk', clk);
+    reset = addInput('reset', reset);
+    requestMask = List.generate(count, (i) => Logic(name: 'requestMask$i'));
+    grantMask = List.generate(count, (i) => Logic(name: 'grantMask$i'));
+    Sequential(clk, [
+      If(reset, then: [
+        for (var g = 0; g < count; g++) requestMask[g] < 1,
+        for (var g = 0; g < count; g++) _grants[g] < 0,
+        for (var g = 0; g < count; g++) grantMask[g] < _requests[g],
+      ], orElse: [
+        /// Added [_grants] to turn off granted bit when a reset on
```

Comment:
The three `///` is for doc comments on public members and functions, use just `//` for other comments.